packages/taquito-rpc/src/types.ts

```diff
@@ -1273,11 +1273,26 @@ export interface TezosGenericOperationError {
   id: string;
 }
 
+export interface TicketUpdates {
+  ticket_token: {
+    ticketer: string;
+    content_type: MichelsonV1Expression;
+    content: MichelsonV1Expression;
+  };
+  updates: {
+    account: string;
+    amount: string;
+  }[]
+}
+export type TicketReceipt  = TicketUpdates
+
 export interface OperationResultTransaction {
   status: OperationResultStatusEnum;
   storage?: MichelsonV1Expression;
   big_map_diff?: ContractBigMapDiff;
   balance_updates?: OperationBalanceUpdates;
+  ticket_updates?: TicketUpdates[];
+  ticket_receipt?: TicketReceipt[];
```

Comment:
I am not sure that showing the 2 properties is the best solution for the users, as we know that `ticket_receipt` will be removed in the protocol M. Let have a discussion about that.